README-vi.md

```diff
@@ -127,34 +127,54 @@ Review the [Contributing Guidelines](CONTRIBUTING.md).
 </p>
 
 * [System design topics: start here](#system-design-topics-start-here)
+* [Các chủ đề thiết kế hệ thống: bắt đầu ở đây](#system-design-topics-start-here)
```

Comment:
I think it's better to be:
[Các chủ đề về thiết kế hệ thống: hãy bắt đầu ở đây](#system-design-topics-start-here)